src/templates/next/components/ArticleCard/ArticleCard.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import { Meta, StoryFn } from "@storybook/react"
+import ArticleCard from "./ArticleCard"
+import ArticleCardProps from "~/common/ArticleCard"
+
+export default {
+  title: "Next/Components/ArticleCard",
+  component: ArticleCard,
+  argTypes: {},
+  parameters: {
+    themes: {
+      themeOverride: "Isomer Next",
+    },
+  },
+} as Meta
+
+// Template for stories
+const Template: StoryFn<ArticleCardProps> = (args) => <ArticleCard {...args} />
+
+export const Default = Template.bind({})
+Default.args = {
+  lastUpdated: "December 2, 2023",
+  category: "Research",
+  title:
+    "A journal on microscopic plastic and their correlation to the number of staycations enjoyed per millennials between the ages of 30-42, substantiated by research from IDK university",
+  url: "/",
+  description:
+    "We’ve looked at how people’s spending correlates with how much microscopic plastic they consumed over the year. We’ve looked at how people’s spending correlates with how much microscopic plastic they consumed over the year.",
+  image: {
+    src: "https://picsum.photos/500/500",
```

Comment:
nit: I think we shouldn't use a randomly generated image cos otherwise Chromatic will keep saying that there are changes to be reviewed, which is quite annoying.

Reply:
oh ya I just wanted to see how the image would resize responsively, will change back to placeholder

Reply:
[519b473](https://github.com/isomerpages/isomer-components/pull/45/commits/519b4732b365199d9c1ce386a3e458bc0d508908)